docs/api-inference/pricing.md

```diff
@@ -1,30 +1,80 @@
-# Pricing and Rate limits
+# Pricing and Billing
 
-As a HF user, you get monthly credits to run the HF Inference API. The amount of credits you get depends on your type of account (Free or PRO or Enterprise Hub), see table below.
-You get charged for every inference request, based on the compute time x price of the underlying hardware.
+The Inference Providers API is a production-ready service involving external partners and is therefore a paid-product. However, as an Hugging Face user you get monthly credits to run experiments. The amount of credits you get depends on your type of account:
+
+| User Tier                | Included monthly credits           |
+| ------------------------ | ---------------------------------- |
+| Free Users               | subject to change, less than $0.10 |
+| PRO and Enterprise Users | $2.00                              |
```

Comment:
it would be great to have a third column with an estimation on how much I can generate (might be hard)